airgun/views/settings.py

```diff
@@ -16,6 +18,10 @@ class SettingsView(BaseLoggedInView, SearchableViewMixin):
         },
     )
 
+    @SatTab.nested
+    class Email(SatTab):
```

Comment:
I am genuinely puzzled by you resistance to implement the exact version I provided in https://github.com/SatelliteQE/airgun/pull/477#discussion_r406824270 .
Decorator is `@View.nested`. Class should inherit from `SatTab`.

I am going to let it slip this time. After all, SatTab does inherit from View (eventually), and this seems to work. But most importantly, we need to branch off 6.7.z and this is only PR here that I am waiting for.

But in the future, please, pay attention to details like that. Nothing bad will happen if you incorporate changes after hour instead of 15 minutes. But every time a change is requested, it adds a day to entire process.